notebooks/tel_and_site/subsys_req_ver/tma/SITCOMTN-067_Vel_Accel_Jerk_15Nov23.ipynb

```diff
@@ -0,0 +1,413 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #58.        fig.clear()
</pre><p>I suggest adding in line breaks for readability in between chunks of code for subplots</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Done